pkg/config/validation/validation_test.go

```diff
@@ -4317,6 +4317,37 @@ func TestValidateServiceEntries(t *testing.T) {
 			},
 			valid: false,
 		},
+		{
+			name: "repeat target port", in: networking.ServiceEntry{
+				Hosts:            []string{"google.com"},
+				WorkloadSelector: &networking.WorkloadSelector{Labels: map[string]string{"key": "bar"}},
+				Ports: []*networking.Port{
+					{Number: 80, Protocol: "http", Name: "http-valid1", TargetPort: 80},
+					{Number: 81, Protocol: "http", Name: "http-valid2", TargetPort: 80},
```

Comment:
This should not be invalid. Its legal in Kubernetes, for example.

Reply:
> This should not be invalid. Its legal in Kubernetes, for example.

I think too much, it should be consistent with k8s